libraries/type-length-value/src/state.rs

```diff
@@ -455,6 +662,15 @@ pub fn realloc_and_pack_variable_len<V: SplDiscriminate + VariableLenPack>(
     Ok(())
 }
 
+/// Packs a variable-length value into an existing TLV space, where no repeating
+/// discriminators are allowed
+pub fn realloc_and_pack_variable_len_unique<V: SplDiscriminate + VariableLenPack>(
```

Comment:
I don't think this has uniqueness guarantees either, right?
```suggestion
pub fn realloc_and_pack_first_variable_len<V: SplDiscriminate + VariableLenPack>(
```